js/content.js

```diff
@@ -259,9 +265,10 @@ function fadeIn(element) {
                 resolve();
             }
             element.style.opacity = op;
-            element.style.filter = "alpha(opacity=" + op * 100 + ")";
-            op += 0.1;
-        }, timerForFading);
+            element.style.filter = "alpha(opacity=" + op * FADE_PERCENT + ")";
```

Comment:
Instead of incrementing by 0.1 and then multiplying by 100 can you just multiply by 10?

Reply:
Will not work, element.style.opacity needs a number between 0 and 1